### DISCUSSION TITLE
Dead island 2

### DISCUSSION THREAD

**Discussion by WarZcommando** [GoldHEN/GoldHEN_Cheat_Repository#185] [Cheat requests]
Hi guys, dead island 2 for ps4 is out and backported already it's been 3 days, so i've been asking if we can get an shn or json cheat file for the game, many thanks guys on the awesome work you've been doing.

**Comment by kmeps4** [comment]
How many times do we have to tell to people that we on GOLDHEN ORGANIZATION we don't develop Cheats.
We add them in our database as soon a cheat developer release it. Please Check: https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42

**Comment by kmeps4** [comment]
I'm gonna close since this is not a bug... cheat will be I'm the repository as soon a Cheat developer release it.
